### PR TITLE
add missing edget scenario, reset sets to H12

### PR DESCRIPTION
**Pull request by orichters**
## Purpose of this PR

- avoid that #1749 leads to compilation failure
- [Removing the scenario](https://github.com/remindmodel/remind/pull/1749/commits/f0f13999423d6632bdae9aa6053b2b2bd4b9ca5c#diff-6c75ce307bba910a36bd94d57e85cc6ad3ac5dde2d75ee11c406fb21cf397ccb) is not possible as long as it is in the input data
- add some `--partition=priority` to `--qos=priority` calls because the cluster does stupid things and the tests failed without it

## Type of change

- [x] Bug fix 

## Checklist:

- [x] My code follows the [coding etiquette](https://github.com/remindmodel/remind/blob/develop/main.gms#L80)
- [x] I performed a self-review of my own code
- [x] I explained my changes within the PR, particularly in hard-to-understand areas
- [x] I checked that the [in-code documentation](https://github.com/remindmodel/remind/blob/develop/main.gms#L120) is up-to-date
- [x] I adjusted the reporting in [`remind2`](https://github.com/pik-piam/remind2) where it was needed
- [x] I adjusted `forbiddenColumnNames` in [readCheckScenarioConfig.R](https://github.com/remindmodel/remind/blob/develop/scripts/start/readCheckScenarioConfig.R) in case the PR leads to deprecated switches
- [x] All automated model tests pass (`FAIL 0` in the output of `make test`): `[ FAIL 0 | WARN 0 | SKIP 6 | PASS 83 ]`
- [x] The changelog `CHANGELOG.md` [has been updated correctly](https://gitlab.pik-potsdam.de/rse/rsewiki/-/wikis/Standards-for-Writing-a-Changelog)

### CODE REVIEW

File: core/sets.gms [hunk 1]
```diff
@@ -762,131 +763,123 @@ sets
...
    regi_group(ext_regi,all_regi) "region groups (regions that together corresponds to a H12 region)"
       /
-        GLO.( OAS,ENC,NES,MEA,SSA,LAM,REF,CAZ,EWN,ECS,CHA,ESC,ECE,FRA,DEU,UKI,NEN,IND,JPN,ESW,USA )
+        GLO.( LAM,OAS,SSA,EUR,NEU,MEA,REF,CAZ,CHA,IND,JPN,USA )
         LAM_regi .(LAM)
         OAS_regi .(OAS)
         SSA_regi .(SSA)
-        EUR_regi .(ENC,EWN,ECS,ESC,ECE,FRA,DEU,UKI,ESW)
-        NEU_regi .(NES,NEN)
+        EUR_regi .(EUR)
```
Comment (jmuessel) [inline core/sets.gms:778]: Why is this necessary? Same question for the other changes regarding regions

Comment (orichters) [reply core/sets.gms:778]: Well, these settings are automatically adapted by the start scripts. Normally, in this repository, they are saved in the REMIND H12 mode, but Johanna apparently ran some REMIND EU-21 tests at the end and committed the adjusted values. It is no problem, but I still prefer to undo it.

**Review by jmuessel — APPROVED** [review]
